python/neuroglancer/pipeline/task_queue.py

```diff
@@ -183,21 +190,43 @@ def cloud_insertion(api):
         else:
             cloud_insertion(self._api)
 
+        return self
+
+    def wait(self):
+        return self.wait_until_queue_empty()
+
     def wait_until_queue_empty(self):
         self._queue.join()
+        return self
 
     def get(self):
         """
```

Comment:
It would be nice to include in the docstring which kind of information gets return, didn't know about this